docs/project/2024/mara.iubitu/index.md

```diff
@@ -0,0 +1,82 @@
+# Proximity Sensor
+A sensor that measures the distance
+
+:::info 
+
+**Author**: Iubitu Mara \
+**GitHub Project Link**: [link_to_github](https://github.com/UPB-FILS-MA/project-MaraIubitu.git)
+
+:::
+
+## Description
+
+A proximity sensor designed for safety and awareness. It accurately detects nearby objects and alerts users visually and audibly through a progressive buzzer and LCD screen. Ideal for navigating tight spaces or vehicles, it offers comprehensive spatial awareness, reducing the risk of accidents.
+
+## Motivation
+
+While researching for project ideas, I came across a parking sensor and a distance measuring project and I thought it would be interesting to try and combine these two ideas.
+
+## Architecture 
+
+![architecture](architectureMaraIubitu.jpg)
+
+## Log
+
+<!-- write every week your progress here -->
+
+### Week 6 - 12 May
+
+### Week 7 - 19 May
+
+### Week 20 - 26 May
+
+## Hardware
+
+Detail in a few words the hardware used.
+
+### Schematics
+
+Place your KiCAD schematics here.
+
+### Bill of Materials
+
+<!-- Fill out this table with all the hardware components that you might need.
+
+The format is 
+```
+| [Device](link://to/device) | This is used ... | [price](link://to/store) |
+
+```
+
+-->
+
+| Device | Usage | Price |
+|--------|--------|-------|
+| [Rapspberry Pi Pico W](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [35 RON](https://www.optimusdigital.ro/en/raspberry-pi-boards/12394-raspberry-pi-pico-w.html) |
+| [Senzor Ultrasonic de Distanță HC-SR04+]() | The distance sensor | [15 RON](https://www.optimusdigital.ro/ro/senzori-senzori-ultrasonici/2328-senzor-ultrasonic-de-distana-hc-sr04-compatibil-33-v-i-5-v.html?search_query=hc-sr04&results=33) |
+| [Breadboard]() | The breadboard | [10 RON](https://www.optimusdigital.ro/ro/prototipare-breadboard-uri/8-breadboard-830-points.html?search_query=breadboard&results=145) |
+| [Wires]() | The wires | [8 RON](https://www.optimusdigital.ro/ro/fire-fire-mufate/12-set-de-cabluri-pentru-breadboard.html?search_query=breadboard&results=145) |
+| [LCD screen]() | The screen | [15 RON](https://www.optimusdigital.ro/ro/optoelectronice-lcd-uri/62-lcd-1602-cu-interfata-i2c-si-backlight-galben-verde.html?search_query=LCD+1602+cu+Interfata+I2C+si+Backlight+Albastru&results=2) |
+| [Buzzer]() | The buzzer | [2 RON](https://www.optimusdigital.ro/ro/audio-buzzere/12247-buzzer-pasiv-de-33v-sau-3v.html?search_query=buzzer&results=62) |
+| [Resistors]() | The resistors | [15 RON](https://www.optimusdigital.ro/ro/componente-electronice-rezistoare/33-rezistoare-set.html?search_query=rezistente&results=115) |
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
```

Comment:
Do you need this library?

Reply:
No, I got confused with the microcontroller name by accident.